### DISCUSSION TITLE
Get listed on the FMI tools page

### DISCUSSION THREAD

**Discussion by chrbertsch** [OFFIS-mosaik/mosaik#16] [General]
As Mosaik supports import and co-simulation of FMUs according to the FMI FMI standard (https://gitlab.com/mosaik/components/mosaik-fmi) I would like to encourage you to get listed on the FMI tools page https://fmi-standard.org/tools/, see https://github.com/modelica/fmi-standard.org/blob/main/CONTRIBUTING.md#updating-the-tools-list for instructions.

**Comment by schwarzatoffis** [comment]
Thank you for that idea. We just created a pull request for mosaik to be added (https://github.com/modelica/fmi-standard.org/pull/622).
